c/tskit/tables.c

```diff
@@ -5660,6 +5661,10 @@ tsk_table_collection_check_integrity(tsk_table_collection_t *self, tsk_flags_t o
             goto out;
         }
         /* Spatial requirements for edges */
+        if (isfinite(left) == 0 || isfinite(right) == 0) {
```

Comment:
I don't think this is the right condition

Reply:
Oh? Not sure what I should use instead...

Reply:
Ah, no, it's right, it's just a bit confusing that way it's stated. What about 
```
if (! (isfinite(left) && isfinite(right))) 
```
Seems more readable.

Reply:
Sure. Committed as an `--amend`